### PR TITLE
[WIP] 51 ui revamp

### PR DESCRIPTION
**Pull request by saowaluck**
Set up docker compose for local dev, Use postgres for local dev  and 	Fix test unit Tests after using postgresSQL  

Connected to #51 🐼

### CODE REVIEW

**Comment by zkan** [comment]
@saowaluck Let's update the requirements and Dockerfiles to have these:

```
flake8==3.3.0
flake8-print==2.0.2
```

File: zendesk_tickets_machine/tickets/tests/test_views.py [hunk 1]
```diff
@@ -188,21 +188,21 @@ def test_ticket_edit_view_should_render_ticket_form(self):
         self.assertContains(response, expected, status_code=200)
 
         expected = '<select name="created_by" class="form-control" ' \
-            'id="id_created_by">'
-        self.assertContains(response, expected, status_code=200)
-        expected = '<option value="1" selected>Kan</option>'
+            'id="id_created_by">\n  <option value="">---------' \
+            f'</option>\n\n  <option value="{self.agent.id}" ' \
```
Comment (nattyait) [inline zendesk_tickets_machine/tickets/tests/test_views.py]: what is f ah

Comment (zkan) [inline zendesk_tickets_machine/tickets/tests/test_views.py]: Literal String Interpolation (Python 3.6) - https://www.python.org/dev/peps/pep-0498/

File: requirements/local.txt [hunk 2]
```diff
@@ -1,3 +1,3 @@
--r base.txt
-
-flake8==3.2.1
+-r base.txt \
```
Comment (zkan) [inline requirements/local.txt]: Please add one blank line here.

File: zendesk_tickets_machine/tickets/tests/test_views.py (hunk 1 above)
Comment (nattyait) [inline zendesk_tickets_machine/tickets/tests/test_views.py]: Thank you!

**Comment by zkan** [comment]
@saowaluck Cool. Thank you! I'll deploy this on Monday in the morning.

**Pull request merged**